II.pass-tests/ES-6-practice/es6.js

```diff
@@ -1,33 +1,33 @@
-// Turn this function into an arrow function
```

Comment:
Sorry I wasn't in class last week - did you cover ES6? If not then I think this will be upcoming homework - it's fine to do it now, but be aware that there may be less homework for you to do in the future (because you've already done it).

Reply:
We'll cover ES6 in more depth in future classes.

Reply:
Yes, we did not cover it. mmm....

Reply:
In that case, don't worry about the other problems in this file. We'll cover them tomorrow or the week after